### DISCUSSION TITLE
Edit / view mode

### DISCUSSION THREAD

**Discussion by char-mainewang** [twilio-labs/paste#3115] [Office hours] [answered]
### Background

After creating an item (i.e. source, destination, model, journey, etc) in segment via the wizard flow, there are usually a few tabs where customers can go in to edit that item.

for example, here's the flow for creating a journey
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/87334135/227026718-c48c637b-1d2a-4551-bba1-7dcc12a67db5.png">
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/87334135/227026774-49bb291b-26d1-431e-8e40-64082dbcf5b9.png">

and here's where customers can make changes to that journey
<img width="1594" alt="image" src="https://user-images.githubusercontent.com/87334135/227026563-8c4e6de8-9063-4c5b-bd56-60e3ce0a4f32.png">
customer clicks on edit button to edit the journey itself
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/87334135/227027099-7f1d10e2-a792-4c77-a461-574e40d26038.png">
customer can also go to the settings page to configure other settings
<img width="1616" alt="image" src="https://user-images.githubusercontent.com/87334135/227026618-b6b34b1c-794d-421d-a42e-5bb0597bc1db.png">

when does it make sense to have a distinct view / edit mode versus just having a page where customers and edit directly?

### Stage of design

Exploration

### Are you looking for visual design support?

- [X] Yes

**Comment by SiTaggart** [comment, marked as answer]
Hi @char-mainewang 

The recording from office hours can be found here https://www.loom.com/share/ad68f125c26446d0a8c6a4beb480112b and the passcode is found in the #help-design-system slack channel.
